build_ngtf.py

```diff
@@ -18,6 +18,34 @@
 from tools.build_utils import *
 
 
+def version_check(use_prebuilt_tensorflow):
+    # Check pre-reqisites
+    if use_prebuilt_tensorflow:
+        # Check if the gcc version is 4.8
+        if (platform.system() != 'Darwin'):
+            gcc_ver = get_gcc_version()
+            if '4.8' not in gcc_ver:
+                raise Exception(
+                    "Need GCC 4.8 to build using prebuilt TensorFlow\n"
+                    "Gcc version installed: " + gcc_ver)
```

Comment:
We can add more lines here that say. "Please change gcc to 4.8 OR build Tensorflow from source by skipping the use_prebuilt_tensorflow option"